src/store/messages/api.ts

```diff
@@ -40,3 +40,23 @@ export async function editMessageApi(
 
   return response.status;
 }
+export async function getInfo(): Promise<infoUploadResponse> {
+  const response = await get<any>('/upload/info');
+  return response.body;
+}
+
+export async function uploadMedia(uploadInfo: infoUploadResponse, media: File): Promise<any> {
+  const response = await postNative<any>(
+    `${uploadInfo.apiUrl}?timestamp=${uploadInfo.query.timestamp}&signature=${uploadInfo.query.signature}&api_key=${uploadInfo.query.api_key}`
+  ).attach('file', media);
+
+  return response;
+}
+
+export async function uploadFileMessage(channelId: string, media: File): Promise<any> {
+  const response = await postNative<any>(`/${channelId}/message`)
```

Comment:
I think this request needs to be with `post` because it's using the credentials of the authenticated user